### DISCUSSION TITLE
Same grain ID, different Euler angles

### DISCUSSION THREAD

**Discussion by KorayKuzkaya** [mtex-toolbox/mtex#1655] [Ask Anything] [answered]
Dear mtex community,

when I click around inside a grain in my ebsd scan, I get different Euler angles for the same grain ID which are in some cases very different.

I load the grain ID as "grain selected" to calculate the active slip systems for a stress tensor. The loaded grain always has the same Euler angles (as seen in the picture 115°,68°,5°).

How can I make sure that I am using the "correct" euler angles, since the angles have a significant effect on the slip plane orientation?

![grain_selected](https://user-images.githubusercontent.com/50950035/201929834-7874d0ba-7b44-4287-a4ba-679310bf56bc.png)
![grain1](https://user-images.githubusercontent.com/50950035/201929858-06ea14d7-ec0e-4708-8a58-6b1999f0b824.png)
![grain2](https://user-images.githubusercontent.com/50950035/201929860-89e12afe-c7dc-487d-8530-0d2e32547a10.png)
![grain3](https://user-images.githubusercontent.com/50950035/201929865-8899f397-bdaa-45a9-ac81-3f92ad9b0dff.png)

**Comment by kilir** [comment, marked as answer]
Hi,
if you only want to use a single orientation for each grain, you can use `grain.meanOrientation` for your computation instead of each orientation inside the grain. 
Cheers,
Rüdiger
